fe/fe-core/src/main/java/com/starrocks/planner/IcebergEqualityDeleteScanNode.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
```

Comment:
I think if we cannot abstract this to leverage connector API, we are not able to abstract the `ConnectorScanNode` in the future

Reply:
yes, we will do this in the future.